tests/test_google_sheets_standard_bookmarks.py

```diff
@@ -0,0 +1,154 @@
+import datetime
```

Comment:
Why was this test added when we already have a bookmarks test? https://github.com/singer-io/tap-google-sheets/blob/master/tests/test_google_sheets_bookmarks.py

Reply:
In the existing bookmark test, standard assertions were missing and it's totally different from our standard bookmark test, [Link](https://github.com/stitchdata/tap-tester/blob/master/reference/expectations/saas_taps.md#bookmarking-on-field---updatedat-datelastedited-etc-1). So, we added a separate bookmark test that includes all standard assertions rather than updating the existing one.

Reply:
Google sheets does not fit the standard replication method test. It performs a hybrid of incremental and full table replication for each sheet. I don't think this test applies here.

Reply:
Removed standard bookmark test case.